core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -5043,7 +5043,9 @@ protected function listView()
 							$value = (string) $arg !== '' ? $arg : '-';
 						}
 
-						$return .= '<td colspan="' . $colspan . '" class="tl_file_list col_' . $field . ($field == $firstOrderBy ? ' ordered_by' : '') . '">' . $value . '</td>';
+						$colclass = 'col_' . explode(":", $field, 2)[0];
+						
+						$return .= '<td colspan="' . $colspan . '" class="tl_file_list ' . $colclass . ($field == $firstOrderBy ? ' ordered_by' : '') . '">' . $value . '</td>';
```

Comment:
```suggestion
						$colclass = 'tl_file_list col_' . explode(":", $field, 2)[0] . ($field == $firstOrderBy ? ' ordered_by' : '');
						
						$return .= '<td colspan="' . $colspan . '" class="' . $colclass . '">' . $value . '</td>';
```

Personally I would prefer all the classes set in one variable, to make it more readable.

Reply:
Much better. I just didnt want to change too much.